src/main/java/de/tum/in/www1/artemis/web/rest/TeamResource.java

```diff
@@ -431,4 +473,17 @@ public TeamResource(TeamRepository teamRepository, TeamService teamService, Team
         course.setExercises(exercises);
         return ResponseEntity.ok(course);
     }
+
+    /**
+     * Sends team assignments updates
+     *
+     * @param exercise Exercise which students will receive team update
+     * @param teams    Teams of exercise
+     */
+    private void sendTeamAssignmentUpdates(Exercise exercise, List<Team> teams) {
+        // Send out team assignment update via websockets
+        Map<String, List<StudentParticipation>> participationsMap = participationService.findByExerciseIdWithEagerSubmissionsResult(exercise.getId()).stream()
+                .collect(Collectors.toMap(StudentParticipation::getParticipantIdentifier, List::of, (a, b) -> Stream.concat(a.stream(), b.stream()).collect(Collectors.toList())));
+        teams.forEach(team -> teamWebsocketService.sendTeamAssignmentUpdate(exercise, null, team, participationsMap.getOrDefault(team.getParticipantIdentifier(), List.of())));
```

Comment:
are we sure that we want to send one update per team?
I would appreciate some inline comments here as well

Reply:
Maybe one update for all imported teams would be enough?

Reply:
I actually did not change this part. I have just used what was used in importing from another exercise.

Reply:
If I'm not completely wrong, this update targets the team, not the instructor, so it can not be replaced by a single message to the instructor.

Reply:
Yes, it sends update to each student.